core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -17,25 +17,18 @@
  * necessary to have a fully-qualified URL.
  */
 
-oppia.factory('UrlInterpolationService', ['alertsService',
-    function(alertsService) {
-  // http://stackoverflow.com/questions/203739
-  var _isString = function(value) {
-    return (typeof value === 'string') || (value instanceof String);
-  };
-
+oppia.factory('UrlInterpolationService', [
+    'alertsService', 'utilsService', function(alertsService, utilsService) {
   var validateResourcePath = function(resourcePath) {
     if (!resourcePath) {
-      alertsService.fatalWarning(
-        'Empty path passed in method.');
+      alertsService.fatalWarning('Empty path passed in method.');
     }
 
     var RESOURCE_PATH_STARTS_WITH_FORWARD_SLASH = /^\//;
     // Ensure that resourcePath starts with a forward slash.
     if (!resourcePath.match(RESOURCE_PATH_STARTS_WITH_FORWARD_SLASH)) {
       alertsService.fatalWarning(
-        'Path must start with \'\/\': \'' + new String(resourcePath) +
```

Comment:
@AllanYangZhou I couldn't find any reason why `new String(...)` is being used here and so removed all occurrences of `new String` from this file. I lean towards removing them from utilsServiceSpec.js as well, since it doesn't seem to fulfill any purpose. wdyt ?